app/controllers/concerns/translatable.rb

```diff
@@ -25,7 +25,7 @@ def delete_translations
     end
 
     def enabled_translations
-      params.fetch(:enabled_translations)
+      params.fetch(:enabled_translations, {})
```

Comment:
Giving a default value to method fetch here, otherwise it will raise an exception in some cases when `:enabled_translations` is not found in params, for example in this test
`spec/features/admin/poll/polls_spec.rb:96`

Reply:
I think it's already fixed by 4d238c5 (which we've merged two minutes ago).